blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -188,38 +188,38 @@ const Results = ({
 	const onReadMoreClick = useCallback(() => {
 		setQueryOffset((oldOffset) => oldOffset + configuredSize);
 	}, [configuredSize, setQueryOffset]);
-
 	return viewableElements?.length > 0 && !isServerSideLazy ? (
-		<div className="results-list-container">
-			{viewableElements.map((element, index) => (
-				<ResultItem
-					key={`result-card-${element._id}`}
-					ref={elementRefs[index]}
-					arcSite={arcSite}
-					element={element}
-					imageProperties={imageProperties}
-					placeholderResizedImageOptions={placeholderResizedImageOptions}
-					showByline={showByline}
-					showDate={showDate}
-					showDescription={showDescription}
-					showHeadline={showHeadline}
-					showImage={showImage}
-					showItemOverline={showItemOverline}
-					targetFallbackImage={targetFallbackImage}
-				/>
-			))}
-			{isThereMore && (
-				<div className="see-more">
-					<Button
-						ariaLabel={phrases.t("results-list-block.see-more-button-aria-label")}
-						buttonStyle={BUTTON_STYLES.PRIMARY}
-						buttonTypes={BUTTON_TYPES.LABEL_ONLY}
-						onClick={onReadMoreClick}
-						text={phrases.t("results-list-block.see-more-button")}
+		<Stack gap="1rem">
```

Comment:
The `gap` should be set in the Sass file not inline if possible

Reply:
updated this